### DISCUSSION TITLE
Edit mode not closed on TAB in last cell of p-table

### DISCUSSION THREAD

**Discussion by AlWoSp** [primefaces/community#518] [PrimeNG]
I guess, this is a bug:
When you press the TAB key while editing the last editable cell of the PrimeNG Table, focus leaves the table, but the cell stays in edit mode.
This can be easily reproduced in the demo for [Cell Edit](https://primeng.org/table#cell-edit) on the PrimeNG site itself.
In this screenshot
![image](https://github.com/primefaces/community/assets/119579533/c94e5a70-ae52-41b8-9642-c11dd76d26ed)
I marked the focus being on the "<>" button below the table, but the cell above still has the editor open ...

**Comment by cetincakiroglu** [comment]
Thanks for reporting, we've created an issue for the problem https://github.com/primefaces/primeng/issues/14221
